### ISSUE TITLE
Failing to init on Ubuntu with 0.3.2-hotfix

### ISSUE THREAD

**Issue by Megalepozy** [vapor-ware/ksync#242]
It seems like issue #237, tried the steps there but it didnt work

$ ksync init
==== Preflight checks ====
Cluster Config                              ✓ 
Cluster Connection                          ✓ 
Cluster Version                             ✓ 
Cluster Permissions                         ✓ 

==== Cluster Environment ====
Adding ksync to the cluster                 ✓ 
Waiting for pods to be healthy              ✓ 

==== Postflight checks ====
Cluster Service                             ✓ 
Service Health                              ✓ 
Service Version                             ✘ 
↳	There is a mismatch between the local version (0.3.2-hotfix) and the cluster ().

Run 'ksync init --upgrade' to fix.
FATA[0000]                  

Running ksync init --upgrade does not work nor does the clean and init (with image tag) commands

_Originally posted by @Megalepozy in https://github.com/vapor-ware/ksync/issues/237#issuecomment-429760082_

**Comment by timfallmk** [comment]
@Megalepozy Actually this is #162 . However the steps are documented further down in the #237 thread as you mentioned.

*tl;dr* `ksync init --remote --image=vaporio/ksync:0.3.2-hotfix`

**Comment by timfallmk** [comment]
@Megalepozy The problem is that remote DaemonSet is not pulling down the correct image (hence the lack of a version number). You're local client binary is correct, but you need to specify the remote version to use.

**Comment by Megalepozy** [comment]
@timfallmk ahh kewl, thanks a lot for ur quick help and awesome project! :)
